process/block/argsProcessor.go

```diff
@@ -0,0 +1,37 @@
+package block
```

Comment:
As to be consistent, should rename the file and structs to argProcessor (no s)

Reply:
Done.